electron/src/lib/openGraph.test.main.ts

```diff
@@ -103,6 +130,6 @@ describe('openGraph', () => {
   it('saves cookies on requests', async () => {
     const cookieText = 'my-cookie';
     const result = await cookieRequest(cookieText);
-    assert.strictEqual(result.config.headers['Cookie'], cookieText);
+    assert.strictEqual(result.config.headers.Cookie, cookieText);
```

Comment:
This won't work with TypeScript as `Cookie` is private and should not be accessed outside of tests.

Reply:
`headers` is `any` in that particular case.. so there are no errors.